website/docs/language/settings/backends/configuration.mdx

```diff
@@ -35,6 +43,18 @@ There are some important limitations on backend configuration:
 - A configuration can only provide one backend block.
 - A backend block cannot refer to named values (like input variables, locals, or data source attributes).
 
+### Credentials and Sensitive Data
+
+Backends store state in a remote service, which allows multiple people to access it. Accessing remote state generally requires access credentials, since state data contains extremely sensitive information.
+
+!> **Warning:**  We recommend using environment variables to supply credentials and other sensitive data. If you use `-backend-config` or hardcode these values directly in your configuration, Terraform will include these values in both state and plan files. This can leak sensitive credentials.
+
+Terraform writes the backend configuration in plain text in two separate files.
+- The `.terraform/terraform.tfstate` file contains the backend configuration for the current working directory.
+- All plan files capture the information in `.terraform/terraform.tfstate` at the time the plan was created. This helps ensure Terraform is applying the plan to correct set of infrastructure.
+
+Configuration stored in a plan file takes precedence over other configuration. This may cause authentication issues if credentials change between creating and applying plans. For example, this can occur in automated environments where credentials are scoped to the duration of a job.
```

Comment:
I wonder if this paragraph is currently assuming too much about what the reader is aware of at this point in their learning, talking about "a plan file" without any other context.

Perhaps I'm overthinking it, but I might consider the following in the hope of spelling out a bit more clearly what we're talking about:

> When applying a plan that you previously saved to a file, Terraform will use the backend configuration stored in that file instead of the current backend settings. If that configuration contains time-limited credentials then they may expire before you finish applying the plan. Pass credentials using environment variables when you need to use different values between the plan and apply steps.

Reply:
you're not overthinking it! This is an excellent point, and I"m adding this to the docs :-) Thank you!